#! /bin/sh /usr/share/dpatch/dpatch-run
## use_system_jars.dpatch by Miguel Landaeta <miguel@miguel.cc>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Configure build to use jar installed on system.

@DPATCH@
diff -urNad httpunit-1.7+dfsg~/build.xml httpunit-1.7+dfsg/build.xml
--- httpunit-1.7+dfsg~/build.xml	2008-05-20 16:25:36.000000000 +0000
+++ httpunit-1.7+dfsg/build.xml	2009-03-04 21:19:43.000000000 +0000
@@ -41,7 +41,7 @@
     <property name="website.dir" value="site" />
     <property name="quicksite.dir" value="../quicksite/src" />
     <property name="lib.dir" value="lib" />
-    <property name="jars.dir" value="jars" />
+    <property name="jars.dir" value="/usr/share/java" />
 
     <property name="docs.dir" value="doc" />
     <property name="build.dir" value="build" />
@@ -302,10 +302,12 @@
         </fileset>
     </copy>
     <mkdir dir="${archive.dir}/jars"/>
+    <!--
     <copy todir="${archive.dir}/jars">
         <mapper type="flatten"/>
         <fileset refid="distributed.jars" />
     </copy>
+    -->
 </target>
 
 
